Manifest.toml

```diff
@@ -0,0 +1,118 @@
+[[Base64]]
+uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+
+[[BinaryProvider]]
+deps = ["Libdl", "Pkg", "SHA", "Test"]
+git-tree-sha1 = "2883bc1b389e3d57a134796c39c788db3f298cef"
+uuid = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+version = "0.4.0"
+
+[[CodecZlib]]
+deps = ["BinaryProvider", "Compat", "Libdl", "Test", "TranscodingStreams"]
+git-tree-sha1 = "c59200e54ffb3eadb796d7fabbdc277f31b7e607"
+repo-rev = "master"
+repo-url = "https://github.com/bicycle1885/CodecZlib.jl.git"
+uuid = "944b1d66-785c-5afd-91f1-9de20f533193"
+version = "0.4.3+"
```

Comment:
Also currently I have used master branch of CodecZlib package to avoid CI build issues. Which I guess isn't good when releasing a package right? Currently MsgPack and CodecZlib package doesn't have a release that support Julia v0.7/v1.0. Now that Julia v1.0 is released as stable release, maybe we should wait and release it once the dependencies are resolved?